### PR TITLE
Update RELEASES.md to include clippy-preview

### PR DESCRIPTION
**Pull request by XAMPPRocky**
[Rendered](https://github.com/rust-lang/rust/blob/Aaronepower-patch-1/RELEASES.md#misc)

r? @Mark-Simulacrum

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ Misc
   will demote `deny` and `forbid` lints to `warn`.
 - [`rustc` and `rustdoc` will now have the exit code of `1` if compilation
   fails, and `101` if there is a panic.][52197]
+- [Added a preview of clippy has been made available through rustup][51122]
```
Comment (phansch) [inline RELEASES.md]: I think this was supposed start with "A preview of Clippy [...]"

**Comment by kennytm** [comment]
@bors rollup

Beta-nominating as part of 1.29's release note.

**Comment by bjorn3** [comment]
@Aaronepower why did you create this branch on rust-lang/rust instead of your personal fork? (https://github.com/rust-lang/rust/tree/Aaronepower-patch-1)

**Comment by Aaronepower** [comment]
@bjorn3 Whoops, that was not intentional I created the pull request through GitHub which before I was a team member would put it on my personal fork, apparently that behaviour changed I've deleted the branch now.

**Comment by bjorn3** [comment]
:) noticed it when I pulled

**Pull request merged**